packages/zone.js/lib/common/events.ts

```diff
@@ -110,6 +110,10 @@ export function patchEventTarget(
     if (task.isRemoved) {
       return;
     }
+    const options = task.options;
+    if (typeof options === 'object' && options?.passive === true) {
```

Comment:
This check is repeated multiple times in this file, I think it'd be great to extract it to a separate function (which would also act as a self-documenting code), for example:

```
function isPassiveEventListener(options: {}): boolean {
  return typeof options === 'object' && options?.passive === true;
}
```
after that the code may look like:
```suggestion
    if (isPassiveEventListener(options)) {
```

Reply:
Thanks @AndrewKushnir , sure, updated.